tests/test_api_endpoints.py

```diff
@@ -489,4 +492,42 @@ def test_systems_mock_rbac_returns_no_groups(
         mock_rbac(get_rbac_mock_file("mock_rbac_returns_no_groups.json"), mocker)
         response = client.get('/api/ros/v1/systems', headers={"x-rh-identity": auth_token})
         assert response.status_code == 200
+        assert response.json["meta"]["count"] == 4
+
+
+def test_systems_mock_rbac_returns_multiple_inventory_hosts_read(
+        auth_token,
+        db_setup,
+        db_create_account,
+        db_create_system,
+        system_with_example_group,
+        system_with_test_group,
+        system_with_foo_group,
+        db_create_performance_profile,
+        create_performance_profiles,
+        mocker):
+    with app.test_client() as client:
+        mock_enable_rbac(mocker)
+        mock_rbac(get_rbac_mock_file("mock_rbac_returns_multiple_read_permissions.json"), mocker)
```

Comment:
@upadhyeammit shall we add a comment here mentioning that `response contains group ids of example-group and food-group`?